### PR TITLE
Add a "Quick Start" guide.

### PR DESCRIPTION
**Pull request by spencercarli**
The goal of this is to make getting started with the library easier so it 

- Adds a quick start guide
- Make the intro contain links to useful resources
- Moves the original intro to Guide-Basic-Example.md
- Widens the docs because they were very cramped with Snacks included

### CODE REVIEW

File: docs/guides/Guide-Basic-Example.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+# This will start a development server for you and print a QR code in your terminal.
+```
+
+Verify that you can successfully see the bare sample app run on iOS and/or Android:
+
+```phone-example
+bare-project
+```
+
+We want to share code on iOS and Android, so let's delete the contents of `index.ios.js` and `index.android.js` and replace it with `import './App';` - after which, we need to create the new file for our app implementation, `App.js` (if you used `create-react-native-app` this has been already done)
```
Comment (matthamil) [inline docs/guides/Guide-Basic-Example.md]: `index.ios.js` and `index.android.js` were deprecated with the latest React Native release https://github.com/facebook/react-native/releases/tag/v0.49.0

File: docs/guides/Guide-Basic-Example.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+          onPress={() => navigate('Chat')}
+          title="Chat with Lucy"
+        />
+      </View>
+    );
+  }
+}
+```
+
+(*don't forget to import View and Button from react-native: * `import { AppRegistry, Text, View, Button } from 'react-native';`)
```
Comment (kelset) [inline docs/guides/Guide-Basic-Example.md]: I think the second \*, before the import, needs to be attached to : or it won't be italic (and this is probably a mistake I did when I updated this file) XD

**Comment by kelset** [comment]
Btw can we make sure to change the text in the Screen show to something which makes it easy to understand that it's simply a Text and not related to the components? I mean, to avoid situations like this #248

File: docs/guides/Guide-Intro.md [hunk 3]
```diff
@@ -1,235 +1,23 @@
...
-}
-```
-
-Now you can see the name when you navigate to the Chat screen. Try changing the `user` param in `HomeScreen` and see what happens!
-
-```phone-example
-first-navigation
-```
+4. [Community Contributions](https://github.com/react-community/react-navigation#community-contributions)
+With the flexibility of React Navigation we won't be able to cover ever possible situation, but another developer may have! Browse our list of community contributions to find topics that may answer your questions.
```
Comment (kelset) [inline docs/guides/Guide-Intro.md]: I think here you meant "...cover everY possible..."

File: docs/guides/Guide-Quick-Start.md [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+    screen: ProfileScreen,
+  },
+});
+
+export default RootTabs;
+```
+
+Getting there! Now let's explicity set a label and icon for the tab bar.
+
+> We'll be using `react-native-vector-icons` in the example. If you don't have it installed in your project already please do so.
```
Comment (kelset) [inline docs/guides/Guide-Quick-Start.md]: Maybe link the to the react native vector icons here.

File: docs/guides/Guide-Quick-Start.md [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+
+This will ensure the `tabBarLabel` is consistent (important when using nested navigators) and it will set a `tabBarIcon`. This icon will only be visible on iOS by default given the tab bar component used, which aligns with standard design patterns on Android.
+
+You can view the complete finished code below:
+
+<div class="snack" data-snack-id="BJZ2GVVpb" data-snack-platform="ios" data-snack-preview="true" data-snack-theme="light" style="overflow:hidden;background:#fafafa;border:1px solid rgba(0,0,0,.16);border-radius:4px;height:505px;width:100%"></div>
+
+## Creating a DrawerNavigator
+
+To get started with `TabNavigator` first import and create a new `RootTabs` component.
```
Comment (kelset) [inline docs/guides/Guide-Quick-Start.md]: I guess you meant DrawerNavigator & RootDrawer ;)

File: docs/guides/Guide-Basic-Example.md [hunk 6]
```diff
@@ -0,0 +1,235 @@
+# Hello Mobile Navigation
+
+Let's use React Navigation to build a simple chat application for Android and iOS.
```
Comment (kelset) [inline docs/guides/Guide-Basic-Example.md]: I know this is probably stupid, but maybe say something like `simple chat-like application` is better, to make sure they won't expect an actual chat to be done in the tutorial XD

File: docs/guides/Guide-Basic-Example.md (hunk 1 above)
Comment (kelset) [reply docs/guides/Guide-Basic-Example.md]: That said maybe rephrasing saying something like `(in older versions, you had two separate files, index balb alb alba and bla bla bla)`

**Comment by spencercarli** [comment]
@kelset @matthamil man am I glad for these reviews haha great catches! Made the requested changes 😄

**Pull request merged**